importer/src/cli.ts

```diff
@@ -0,0 +1,21 @@
+import { program } from 'commander';
+import { run } from './cli-run.js';
+import { IMPORTER_VERSION } from './version.js';
+
+program
+  .name('xata-importer')
+  .description('Importer utility for xata.io')
+  .version(IMPORTER_VERSION)
+  .argument('<file>', 'File containing the data to import. Pass "-" to pipe content instead.')
+  .option('--table <table>', 'The table in which to import the data.')
+  .option('--columns <columns>', 'List of column names.')
+  .option('--types <types>', 'List of column names.')
+  .option('--noheader', 'Specify that the CSV file has no header.')
+  .option('--format <format>', 'The format of the input source.', 'csv')
+  .option('--create', "Whether the table or columns should be created if they don't exist.", false)
```

Comment:
Not sure I get the ``--create`` flag. If I don't pass it, it errors me out. If I do pass it still asks me for confirmation.

In case you don't pass it I would ask interactively and have the flag to not ask.

<img width="758" alt="Screenshot 2022-05-04 at 08 20 19" src="https://user-images.githubusercontent.com/2181866/166631559-8b22a439-3714-474a-9bb1-ebf575bd66f5.png">

Reply:
Yeah, although I would make it always ask and maybe have a interactive/non-interactive flag that disables inquirer everywhere and fails if the flags for ``create`` or ``force`` are not provided. WDYT?